src/main/java/com/puppycrawl/tools/checkstyle/gui/ParseTreeTablePModel.java

```diff
@@ -242,7 +242,7 @@ public int getIndexOfChild(Object parent, Object child) {
      * @return true if editable
      */
     public boolean isCellEditable(int column) {
-        return getColumnClass(column).equals(ParseTreeTablePModel.class);
+        return getColumnClass(column).equals(ParseTreeTableModel.class);
```

Comment:
Looks like you found dead and weird code.
do "return false", and recheck GUI that it works fine.